@navikt/core/tokens/config/tests/validate-docs.test.ts

```diff
@@ -3,10 +3,10 @@ import DsTokens from "../../dist/tokens-cjs.js";
 import docs from "../../docs.json";
 import kebabCase from "../kebabCase.js";
 
-const flatten = Object.values(docs).reduce(
-  (old, val) => [...old, ...val],
-  [] as any,
-);
+const flatten = Object.values(docs).reduce((old, val) => {
+  old.push(...val);
+  return old;
+}, [] as any);
```

Comment:
Have you considered using `flat()` instead? Don't know about the performance, but I think it's easier to read. Looks like there are other cases in this PR it could be used, too.

Reply:
Fixed 🚀

Reply:
Checking for other instances where it can be used also 👍